defaults/main.yml

```diff
@@ -446,7 +453,7 @@ tmp_mount_file:
   Debian: /usr/share/systemd/tmp.mount
 tmp_mount_options:
   RedHat: mode=1777,strictatime,noexec,nodev,nosuid
-  Debian: mode=1777,strictatime,nodev,nosuid
+  Debian: mode=1777,strictatime,noexec,nodev,nosuid
```

Comment:
Setting noexec as default for tmp is not a good idea. It will crash apt. This option only make sense if you will never use apt to install a package or update your software. (e.g. a cloud image)

Reply:
Yes that's my usage case. Anyway, I separated the noexec part to a different task and set it to false by default. It's possible to use it but just if you explicitly set ubuntu1804cis_rule_1_1_5 to true